security_monkey/tests/watchers/test_s3.py

```diff
@@ -66,6 +66,3 @@ def test_slurp(self):
         item_list, exception_map = s3_watcher.slurp()
 
         assert len(item_list) == 3  # We created 3 buckets
-
-        mock_s3().stop()
```

Comment:
Noticing the removal of the `stop()`s on the mocks. Is this OK?

Reply:
This was a temporary work around after getting stop exceptions for patches not working.  patches have changed in mock and this needs a little love, but as is, these stops were attempting to stop an already exited patch.